### PR TITLE
Show author's book

### PR DESCRIPTION
**Pull request by NguyenThiHangB**

### CODE REVIEW

File: app/views/authors/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    <% end %>
+  </div>
+</div>
+<div class="row">
+  <ul class="authors">
+    <% @authors.each do |author| %>
+      <li>
+        <span class="user">
+          <%= link_to author.name, author %> -
+          <%= author.brith_year.strftime("%Y") %>
```
Comment (huongnguyenmta) [inline app/views/authors/index.html.erb]: dùng `I18n format time`

File: app/views/authors/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  <div class="col-md-8">
+    <p class="title-favorite"><%= t ".book" %> <%= link_to @author.name %></p>
+    <ol class="favorite">
+      <% @list_books.each do |list_book| %>
+        <li>
+          <%= image_tag list_book.image_url.to_s, class: "gravatar" %>
+          <span class="user"><%= link_to list_book.name, list_book %></span>
+          <i class="timestamp">
+            <%= t ".publish" %>
+            <%= list_book.publish_date.strftime("%d-%m-%Y") %>
```
Comment (huongnguyenmta) [inline app/views/authors/show.html.erb]: dùng `I18n format time`

**Comment by huongnguyenmta** [comment]
commented

**Comment by NguyenThiHangB** [comment]
ready

**Pull request merged**